scripts/check-config.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
```suggestion
#!/usr/bin/env bash

set -euo pipefail
```

Let's follow the best practices here. I'm also adding shellcheck to pre-commit in #16.

Reply:
This would result in immediately aborting the script when the `compgen` or `python3` command fails and less usefull output from my understanding?